src/components/App/index.tsx

```diff
@@ -52,7 +52,9 @@ const Inner = () => {
   const isProjects = activePage === VIEW_TYPES.projects
   const isTeam = activePage === VIEW_TYPES.modites
   //  let isGlobe = listOptions.view === 'globe'
-  let isGlobe
+  let isGlobe: boolean
+  // @todo check this with Mike Kezii about this commit https://github.com/moduslabs/modir/commit/91db7bad10eb499b53ccdf9950b2a6fd890b13ec --> ignore temporary
```

Comment:
@Mrkezii I am not sure if get this right, could you please explain this one?

Reply:
Hey, what about it specifically can i help with ? 

As far as i can explain the logic was changed in this [commit ](https://github.com/moduslabs/modir/pull/95/files#diff-df083a776c8512b421952bdfefa3f49d267924da306e81c8ca8659597330e6c7R55)

The main gist is that the variable is used to determine what icon the user sees at the top right corner of the home page. It could be either a globe or a list

Reply:
my question is relative to `isGlobe` is undefined and then next line is on a conditional, that is one issue on lint that I have added ignore